refactor(src/content/developers/docs/gas/index.md): fix some grammar issues #3908 (#4118)
ethereum/ethereum-org-website: src/content/developers/docs/gas/index.md

High gas fees are due to the popularity of Ethereum. Performing any operation on Ethereum requires consuming gas, and gas space is limited per block. Fees include calculations, storing or manipulating data, or transferring tokens, consuming different amounts of "gas" units. As dapp functionality grows more complex, the number of operations a smart contract performs also grows, meaning each transaction takes up more space of a limited size block. Users must offer a higher tip amount to try and outbid other users' transactions if there's too much demand. A higher tip can make it more likely that your transaction will get into the next block.

Gas price alone does not determine how much we have to pay for a particular transaction. To calculate the transaction fee, we have to multiply the gas used by the transaction fee, measured in gwei.

## Initiatives to reduce gas costs {#initiatives-to-reduce-gas-costs}

With the new network upgrades of Ethereum 2.0 (also known as Eth2 or Serenity) should ultimately address some of the gas fee issues, which will, in turn, enable the platform to process thousands of transactions per second and scale globally.

> Layer 2 scaling is a primary initiative to improve gas costs, user experience, and scalability greatly. [More on layer 2 scaling](/developers/docs/scaling/layer-2-rollups/).

Reviewer: Personally felt this read better flipped back:
```suggestion
Layer 2 scaling is a primary initiative to greatly improve gas costs, user experience and scalability. [More on layer 2 scaling](/developers/docs/scaling/layer-2-rollups/).
```
Just 1 opinion though

> The new proof-of-stake model should reduce high power consumption and reliance on specialized hardware. In addition, the new PoS system was introduced on the Beacon Chain. This chain will allow the decentralized Ethereum network to agree and keep the network secure, but avoid high energy use by requiring a financial commitment.

Reviewer: Suggested cleanup:
```suggestion
The new proof-of-stake model, introduced on the Beacon Chain, should reduce high power consumption and reliance on specialized hardware. This chain will allow the decentralized Ethereum network to agree and keep the network secure, while limiting energy consumption by instead requiring a financial commitment.
```

> Anyone with at least 32 ETH can to stake them and become a validator responsible for processing transactions, proposing new blocks to, add to the blockchain and storing data. Users who have less than 32 ETH can join staking pools.

Reviewer: ```suggestion
Anyone with at least 32 ETH can stake them and become a validator responsible for processing transactions, validating blocks, and proposing new blocks to add to the chain. Users who have less than 32 ETH can join staking pools.
```
